packages/lib-utils/src/ws/types.ts

```diff
@@ -0,0 +1,27 @@
+import type { AnyObject } from '@monorepo/common';
+
+type GenericHandler<T = unknown> = (data: T) => void;
+export type OpenHandler = GenericHandler<never>;
+export type CloseHandler = GenericHandler<CloseEvent>;
+export type ErrorHandler = GenericHandler<Event>;
+/**
+ * The WebSocket data can be returned in an object state or in the raw string response passed.
+ *
+ * This is configured through `jsonParse` options flag.
+ * @see WSOptions
```

Comment:
FYI, if `WSOptions` type was imported, you could reference it like so
```
@see {@link WSOptions}
```
but there are times when it shouldn't be imported, so above is OK.